Unix/installbuilder/conf/omiserver.conf

```diff
@@ -33,31 +35,49 @@ httpsport=0
 #NtlmCredsFile=/etc/opt/omi/.creds/ntlm
 
 ##
-## <NICKNAME> -- set the value of nickname.
-##
-#prefix=PATH
-#libdir=PATH
-#bindir=PATH
-#localstatedir=PATH
-#sysconfdir=PATH
-#providerdir=PATH
-#certsdir=PATH
-#datadir=PATH
-rundir=/var/opt/omi/run
-#logdir=PATH
-#schemadir=PATH
-#schemafile=PATH
-pidfile=/var/opt/omi/run/omiserver.pid
-logfile=/var/opt/omi/log/omiserver.log
-registerdir=/etc/opt/omi/conf/omiregister
-pemfile=/etc/opt/omi/ssl/omi.pem
-keyfile=/etc/opt/omi/ssl/omikey.pem
-#agentprogram=PATH
-#serverprogram=PATH
-#includedir=PATH
-configfile=/etc/opt/omi/conf/omiserver.conf
+## This is the setting for omi related directory and file path, the following values are default values, and we don't support modifing the path.
```

Comment:
Limit lines to 80 bytes or less in configuration files please.

Reply:
Run-on sentence.

Reply:
Removed commented default values section.